change/@microsoft-teams-js-4f7e8cc9-56bd-4cc3-994e-d0dcac0805c5.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "modify more typedocs around actions",
```

Comment:
>modify more typedocs around actions [](http://example.com/codeflow?start=14&length=35)

Since this goes into the public changelog, we like to use a consistent tense (past tense). Consider:

`Clarified documentation around action interfaces`

Reply:
```suggestion
  "comment": "Clarified documentation around action interfaces",
```